tests/Unit/Translation/TranslatorTest.php

```diff
@@ -229,4 +229,20 @@ public function testLoadingTranslationList()
         $translationValues = $translations[0]->getTranslations();
         $this->assertArrayNotHasKey('fr', $translationValues);
     }
+
+    public function testCacheGetsInvalidatedOnSave() {
```

Comment:
```suggestion
    public function testCacheGetsInvalidatedOnSave(): void
    {
```

Reply:
it is for 10.5 :) We will do it for 11.

Reply:
You can it also for new methods in 10.5. Or why not?

Reply:
I want to keep the files consistent. Adding it for just one method feels kinda wrong.

Reply:
@mattamon Now it is in branch 11.x, so you add it there?
https://github.com/pimcore/pimcore/blob/11.x/tests/Unit/Translation/TranslatorTest.php#L232

Reply:
> @mattamon Now it is in branch 11.x, so you add it there? https://github.com/pimcore/pimcore/blob/11.x/tests/Unit/Translation/TranslatorTest.php#L232

Yes, unless you want to  add it? :) Then a PR is highly appreciated :)

Reply:
@mattamon I created a PR, I found also other missing type hints: https://github.com/pimcore/pimcore/pull/15185